static/src/javascripts/projects/common/modules/experiments/tests/tailor-survey.js

```diff
@@ -210,61 +211,42 @@ const handleSurveyResponse = surveyId => {
     });
 };
 
-class TailorSurvey {
-    id: string;
-    start: string;
-    expiry: string;
-    author: string;
-    description: string;
-    audience: number;
-    audienceOffset: number;
-    successMeasure: string;
-    audienceCriteria: string;
-    dataLinkNames: string;
-    idealOutcome: string;
-    variants: Array<Object>;
-    canRun: () => boolean;
-
-    constructor() {
-        this.id = 'TailorSurvey';
-        this.start = '2017-03-07';
-        this.expiry = '2017-05-31';
-        this.author = 'Manlio & Mahana';
-        this.description = 'Testing Tailor surveys';
-        this.audience = 1;
-        this.audienceOffset = 0;
-        this.successMeasure =
-            'We can show a survey on Frontend as decided by Tailor';
-        this.audienceCriteria = 'All users';
-        this.dataLinkNames = 'Tailor survey';
-        this.idealOutcome = '';
-        this.canRun = () =>
-            !config.page.isAdvertisementFeature &&
-            config.page.contentType === 'Article';
-        this.variants = [
-            {
-                id: 'control',
-                test: () => {},
+export const tailorSurvey = new ABTest({
```

Comment:
would it be more readable if `ABTestConfig` was used directly? eg. `const config: ABTestConfig = {...}`, then `new ABTest(config)`? Just wondering if that makes the associated type of the constructor parameter more explicit. Bit more verbose though, just a thought really.